.github/workflows/build.yaml

```
@@ -804,6 +804,46 @@ jobs:
         message-path: build-comment.md
 
+  # Create GKE cluster early — runs in parallel with the entire build.
+  # By the time images are pushed (~30 min), the cluster is already ready.
+  # Only created when 'e2e-gke' label is present on the PR.
+  create-e2e-gke-cluster:
+    if: >-
+      contains(github.event.pull_request.labels.*.name, 'e2e-gke')
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    outputs:
+      cluster-name: ${{ steps.create.outputs.cluster-name }}
```

actionlint (1.7.12)

[error] property "cluster-name" is not defined in object type {}

(expression)